src/wechaty/user/message.py

```diff
@@ -328,7 +328,7 @@ async def mention_list(self) -> List[Contact]:
         """
         log.info('Message mention_list')
         room = self.room()
-        if self.type() != MessageType.Text or room is None:
+        if self.type() != MessageType.MESSAGE_TYPE_TEXT or room is None:
```

Comment:
I believe there are more attribute name errors exist in this file. eg:
`MessageType.Room`, `MessageType.Contact`, `MessageType.Recall` and so on.